### PR TITLE
Stock report cleanup

### PR DESCRIPTION
**Pull request by czue**
follow up from https://github.com/dimagi/commcare-hq/pull/12111#issuecomment-227100392

turns out that `location_id` is properly set in SMS workflows so didn't remove it (it might be safe to remove there as well, but was more than I was interested in taking on just now)

@snopoke cc @gcapalbo easiest commit by commit. :fish:

### CODE REVIEW

File: corehq/apps/commtrack/sms.py [hunk 1]
```diff
@@ -135,7 +136,7 @@ def parse(self, text):
                 "You can no longer use requisitions! Please contact your project supervisor for help"
             ))
 
-        elif self.C.multiaction_enabled and action_keyword == self.C.multiaction_keyword:
+        elif self.commtrack_settings.multiaction_enabled and action_keyword == self.commtrack_settings.multiaction_keyword:
```
Comment (dimagimon) [inline corehq/apps/commtrack/sms.py]: E501 line too long (123 > 115 characters)

**Comment by snopoke** [comment]
:+1: on :white_check_mark:

File: corehq/apps/commtrack/sms.py [hunk 2]
```diff
@@ -135,7 +136,8 @@ def parse(self, text):
                 "You can no longer use requisitions! Please contact your project supervisor for help"
             ))
 
-        elif self.C.multiaction_enabled and action_keyword == self.C.multiaction_keyword:
+        elif (self.commtrack_settings.multiaction_enabled
+                  and action_keyword == self.commtrack_settings.multiaction_keyword):
```
Comment (dimagimon) [inline corehq/apps/commtrack/sms.py]: E127 continuation line over-indented for visual indent

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**